easybuild/easyconfigs/g/GCC/GCC-4.9.3-binutils-2.25.eb

```diff
@@ -25,7 +26,12 @@ sources = [
     'mpc-1.0.2.tar.gz',
 ]
 
-builddependencies = [('binutils', binutilsver)]
+dependencies = [
+    # GCC built on top of (dummy-built) binutils
+    ('GCCcore', gccver, '-binutils-%s' % binutilsver),
```

Comment:
can't this be a build dependency? (but this probably doesn't make any difference since it will be a dependency trough binutils anyway?)